ingestion/src/metadata/ingestion/stage/table_usage.py

```diff
@@ -116,7 +120,7 @@ def stage_record(self, record: QueryParserData) -> None:
                             joins.append(tbl_column_join)
 
                     table_usage_count = TableUsageCount(
-                        table=table,
+                        table=_get_formmated_table_name(table),
```

Comment:
Can you test it out with other connectors and double check that it isn't interfering with other usages